### PR TITLE
Add block size and gas limit to validation

### PR DESCRIPTION
**Pull request by anorth**

### CODE REVIEW

File: src/systems/filecoin_blockchain/struct/block/_index.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ A syntactically valid full block must have:
 
 - all referenced messages syntactically valid,
 - all referenced parent receipts syntactically valid,
+- the sum of the serialized sizes of all messages is no greater than `block.BlockMaxSize`,
```
Comment (sternhenri) [inline src/systems/filecoin_blockchain/struct/block/_index.md]: Does it not include the header ie
```suggestion
- the sum of the serialized sizes of all messages and the header is no greater than `block.BlockMaxSize`,
```

File: src/systems/filecoin_blockchain/struct/block/_index.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ A syntactically valid full block must have:
...
+- the sum of the gas limit of all messages is no greater than `block.BlockGasLimit`.
```
Comment (sternhenri) [inline src/systems/filecoin_blockchain/struct/block/_index.md]: may want to remind reader to include cron-activated messages (i.e. implicit messages) w gas cost

**Comment by sternhenri** [comment]
Also fixing of compilation obviously :b

File: src/systems/filecoin_blockchain/struct/block/_index.md (hunk 2 above)
Comment (anorth) [reply src/systems/filecoin_blockchain/struct/block/_index.md]: I intend to explicitly exclude the implicit messages, which will have large, fixed gas limits.

**Pull request merged**